content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -363,6 +363,38 @@ rules:
   verbs: ["get", "list", "watch"]
 ```
 
+#### Write Access for Endpoints
+
+Kubernetes clusters created before Kubernetes 1.22 include write access to
+Endpoints in the aggregated "edit" and "admin" roles. As a mitigation for
+[CVE-2021-25740](https://github.com/kubernetes/kubernetes/issues/103675), this
+access will not be part of the aggregated roles in newly created clusters using
+Kubernetes 1.22+.
+
+If you want new clusters to retain this level of access in the aggregated roles,
+you can create the following ClusterRole:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    kubernetes.io/description: |-
+      Add endpoints write permissions to the edit and admin roles. This was
+      removed by default in 1.22 because of CVE-2021-25740. See
+      http://issue.k8s.io/103675. This can allow endpoint writers to direct
+      LoadBalancer or Ingress implementations to expose backend IPs that would
```

Comment:
```suggestion
       load balancers or similar components to expose backend IPs that would
```

Reply:
I think it's important to mention Ingress implementations here as not everyone may draw the connection between LB and Ingress.